cypress/e2e/loginlogout.cy.js

```diff
@@ -0,0 +1,27 @@
+import { login } from "../../src/js/api/auth/login";
+
+beforeEach(() => {
+  cy.visit("../../index.html");
```

Comment:
```suggestion
  cy.visit("/");
```

You should be able to use this to access the homepage